editor/plugins/tiles/tile_set_editor.cpp

```diff
@@ -997,6 +997,7 @@ void TileSourceInspectorPlugin::_confirm_change_id() {
 }
 
 bool TileSourceInspectorPlugin::can_handle(Object *p_object) {
+	ERR_FAIL_NULL_V(p_object, false);
 	return p_object->is_class("TileSetAtlasSourceProxyObject") || p_object->is_class("TileSetScenesCollectionProxyObject");
```

Comment:
This might be better handled with:
```suggestion
	return Object::cast_to<TileSetAtlasSourceProxyObject>(p_object) || Object::cast_to<TileSetScenesCollectionProxyObject>(p_object);
```
And remove the above check